### PR TITLE
Add KIC 1.1.0 documentation

### PR DESCRIPTION
**Pull request by rainest**
Add documentation for KIC 1.1.x. This is a small release with only one new user-facing feature (the `konghq.com/snis` annotation), so it's basically just boilerplate and a copy of the 1.0.x docs.

Awaiting netlify preview to confirm I didn't bork the boilerplate or copying the compatibility table column.

### CODE REVIEW

**Comment by rainest** [comment]
Having remembered that there are multiple compat tables and added 1.1.x to all of them, should now be good to go.

File: app/kubernetes-ingress-controller/1.1.x/references/annotations.md [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+
+### konghq.com/snis
+
+> Available since controller 1.1
+
+Sets the `snis` match criteria on the Kong route associated with this Ingress.
+These allow route matching based on the server name indication information sent
+in a client's TLS handshake, for route-attached plugins that execute during the
+certificate phase (for example, [Mutual TLS
+Authentication](https://docs.konghq.com/hub/kong-inc/mtls-auth/)).
```
Comment (lena-larionova) [inline app/kubernetes-ingress-controller/1.1.x/references/annotations.md:326]:
```suggestion
When using route-attached plugins that execute during the certificate 
phase (for example, [Mutual TLS Authentication](/hub/kong-inc/mtls-auth/)), 
the `snis` annotation allows route matching based on the server name 
indication information sent in a client's TLS handshake.
```
Context first, if possible, so the reader knows if they need to keep reading or not.

**Review by lena-larionova — COMMENTED** [review]
Looks like everything works as it should, the new version comes up as latest and the old one appears in the version dropdown. One minor comment to rephrase the `konghq.com/snis` description, otherwise looks good.

**Review by lena-larionova — APPROVED** [review]
LGTM

**Pull request merged**